### ISSUE TITLE
test failed in AT

### ISSUE THREAD

**Issue by hainm** [Amber-MD/cpptraj#537]
Dan,

I've been getting cpptraj failures within AT (tried with my Mac, travis, circleci, ...)
Can you try that? thanks

```
==============================================================
===================== TEST SUMMARY ======================
  0 out of 1 comparisons OK (1 failed, 0 warnings).
  0 out of 0 tests completed with no issues.
=========================================================
make[2]: *** [summary] Error 1
make[2]: Leaving directory `/home/haichit/amber_git/amber/AmberTools/src/cpptraj/test'
make[1]: *** [test.all] Error 2
make[1]: Target `test' not remade because of errors.
make[1]: Leaving directory `/home/haichit/amber_git/amber/AmberTools/src/cpptraj/test'
make: *** [test.cpptraj] Error 2
```

**Comment by hainm** [comment]
bonus failure (from circleci AmberTools test (created from its tarfile via mkrelease_at).

```
  CPPTRAJ: Data filter test.
Error: '../Test_Diffusion/diff_a.xmgr': No such file or directory
Error: Could not read data file '../Test_Diffusion/diff_a.xmgr'.
	1 errors encountered reading input.
Error: Error(s) occurred during execution.
Error: cpptraj exited with status 1
possible FAILURE:  file datafilter.dat does not exist.
==============================================================
**************************************************************
```

**Comment by drroe** [comment]
Should be addressed by #538.

**Comment by hainm** [comment]
thanks
